swagger.js

```diff
@@ -32,100 +32,135 @@ const doc = {
         },
       },
     },
-    TokenizedUser: {
-      user: {
-        name: 'John Doe',
-        userId: '64ef5885c2a489217c571253',
-        role: { '@enum': ['user', 'admin'] },
-      },
-    },
-    UserWithAlbums: {
-      $id: '64d6ca92a15d2e18ab96a2a3',
-      $name: 'John Doe',
-      $username: 'johndoe',
-      $email: 'john@example.com',
-      $role: { '@enum': ['user', 'admin'] },
-      profileImage: {
-        url: 'https://example.com/profile-image.jpg',
-      },
-      creditCardInfo: {
-        hashedNumber: '1234567890',
-        expiry: '12/24',
-        preferredPaymentOption: {
-          '@enum': ['credit card', 'paypal', 'google pay'],
+    definitions: {
```

Comment:
@DYA13 we still need to delete line 35 and line 70 so that we don't have this extra `definitions` block in here surrounding our Order object.

Reply:
Akos thank you for detailed explanation ❤️

Reply:
I have added New Order, at the moment it is like an object, as it was before. If I need to change for an array, tell me please.